최창익/app/src/main/java/com/mash_up/mvvmstudy/BindingAdapter.kt

```diff
@@ -0,0 +1,17 @@
+package com.mash_up.mvvmstudy
+
+import android.widget.ImageView
+import androidx.core.content.ContextCompat
+import androidx.databinding.BindingAdapter
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.RequestOptions
+
+@BindingAdapter("app:imageUrl")
```

Comment:
혹시 app: prefix를 붙이신 이유가 있을까요! 제거해도 xml에서 imageUrl로 접근할 수 있더라구요
```suggestion
@BindingAdapter("imageUrl")
```

Reply:
앗 안붙여도 되는군요! 제거하는게 더 좋을 것 같습니다ㅎㅎ